docs/data/date-pickers/custom-field/BrowserV6SingleInputRangeField.tsx

```diff
@@ -103,9 +119,11 @@ const BrowserSingleInputDateRangeField = React.forwardRef(
       ),
     };
 
-    const fieldResponse = useSingleInputDateRangeField<Dayjs, typeof textFieldProps>(
-      textFieldProps,
-    );
+    const fieldResponse = useSingleInputDateRangeField<
+      Dayjs,
+      false,
```

Comment:
I did not add any default value to `TEnableAccessibleFieldDOMStructure` except on widely used top level components (the components and their prop interfaces) because I'm slightly worried that we forget to pass the right generic value if TS does not complain when we don't.
Even with just those "few" interfaces that have a default value I ended up with a lot of problems.

IMHO it's not a bad thing that people creating custom UI need to pass generics, that way they know that a new DOM structure exists and maybe they should not do their deep customization with the legacy approach.

Reply:
Ok, that makes sense. Indeed, having the default overspecified can bite us. 🙈